core/src/test/scala/unit/kafka/utils/JaasTestUtils.scala

```diff
@@ -166,7 +187,7 @@ object JaasTestUtils {
           keyTab = keytabLocation.getOrElse(throw new IllegalArgumentException("Keytab location not specified for GSSAPI")).getAbsolutePath,
           principal = KafkaServerPrincipal,
           debug = true,
-          serviceName = Some("kafka")).toJaasModule
+          serviceName = serviceName).toJaasModule
```

Comment:
@rajinisivaram Within the current tests, serviceName was always provided so using the option looked to me as speculative generality